lua/telescope/_extensions/ui-select.lua

```diff
@@ -28,11 +28,19 @@ return require("telescope").register_extension {
               client_name = 0,
             }
             for idx, item in ipairs(items) do
-              local client = vim.lsp.get_client_by_id(item[1])
+              local client, title
+              if vim.fn.has "nvim-0.10.0" then
```

Comment:
ah sorry, i dont really like `vim.fn.has` can you rather use `vim.version().minor`

thanks :)

Reply:
I believe `vim.version` has been added in 0.9 (https://neovim.io/doc/user/news-0.9.html), so, if we'll use it we should also bump version in README, as it says:
> requires latest nvim 0.7 or newer nightly version

BTW, in the https://neovim.io/doc/user/lua.html#version-range there is example how to compare versions, so I guess the proper way of comparing version would be:
```lua
local r = vim.version.range('>0.10')
if r:has(vim.version()) then
``` 
but, `:has` marked as private which is confusing. 

To keep backward compatibility with 0.7 and 0.9, we may write something:
```
vim.version and vim.version() >= vim.version.parse "0.10" then
```

Reply:
yeah good point, i forgot that `vim.version` is a pretty recent addition :sweat_smile: